vendor/github.com/hashicorp/go-discover/go.mod

```diff
@@ -56,6 +56,7 @@ require (
 	github.com/tencentcloud/tencentcloud-sdk-go v3.0.83+incompatible
 	github.com/tent/http-link-go v0.0.0-20130702225549-ac974c61c2f9 // indirect
 	github.com/vmware/govmomi v0.18.0
+	golang.org/x/crypto v0.0.0-20181029021203-45a5f77698d3
```

Comment:
There's no other vendor or go.mod changes in this PR. Is this file supposed to be here?